model/Modifiable.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+/**
+ * Interface Modifiable.
+ */
+interface Modifiable {
+
```

Comment:
New interface. Adding to a parent class did not work, as well as a trait, because the two modifiables that we have (Vocabulary and Concept) each have a different way to get the modified date, and whether 304s are enabled or not.